android/app/src/main/cpp/app/organicmaps/editor/OpeningHours.cpp

```diff
@@ -271,7 +271,7 @@ Java_app_organicmaps_editor_OpeningHours_nativeSetIsFullday(JNIEnv * env, jclass
 
 JNIEXPORT jobject JNICALL
 Java_app_organicmaps_editor_OpeningHours_nativeSetOpeningTime(JNIEnv * env, jclass clazz,
```

Comment:
nit: if unused `clazz` is removed, most of these declarations can nicely fit into one line instead of two lines.